File: Even.java
import java.util.Scanner;
public class Even{
public static void main(String... args){

Scanner input = new Scanner(System.in);

System.out.println("enter a number  ");
int number = input.nextInt();

boolean result = Kata.isEven(number);
System.out.print("is " + number + " an even number? " + result);

}
}

File: even.c
#include <stdio.h>
#include <stdbool.h>

bool isEven(int num) {
    return num % 2 == 0;
}

int main() {
    int num;
    printf("Enter an integer: ");
    scanf("%d", &num);
    
    if (isEven(num)) {
        printf("%d is even.\n", num);
    } else {
        printf("%d is odd.\n", num);
    }
    
    return 0;
}